orienteer-core/src/main/java/org/orienteer/core/util/CommonUtils.java

```diff
@@ -85,6 +85,10 @@ public static final CharSequence escapeStringForJSON(CharSequence content) {
 				.collect(Collectors.toList());
 	}
 
+	public static List<ODocument> getDocuments(List<OIdentifiable> identifiables) {
+		return mapIdentifiables(identifiables, d -> d);
```

Comment:
OIdentifiable can lead to null. Such records should be excluded.

Reply:
Fixed in 3a69ccc